ansible/group_vars/all

```diff
@@ -145,10 +146,22 @@ kafka:
       retentionBytes: 1073741824
       retentionMS: 300000
 
+kafka_connect_string: "{% set ret = [] %}
```

Comment:
This multiline script leads to a string with whitespaces at the beginning of the property that is mapped into the container. 
```
"KAFKA_HOSTS=     192.168.99.100:9092,192.168.99.100:9093",
"ZOOKEEPER_HOSTS=    192.168.99.100:2181,192.168.99.100:2182",
```
If we put it in one line by removing the spaces between braces, like `}{` the setting looks correct. Another option might be to run something like a trim() on the resulting string after the joinL